src/sentry/tasks/post_process.py

```diff
@@ -22,6 +23,16 @@
 logger = logging.getLogger('sentry')
 
 
+def _get_service_hooks(project_id):
+    from sentry.models import ServiceHook
+    cache_key = 'servicehooks:1:{}'.format(project_id)
```

Comment:
is `1` a servicehooks version number, or just something you can bump to invalidate the cache quickly?

Reply:
@alex-hofsteede both, but just internal to the cache key